### PR TITLE
chore: update pytest testpaths config

### PR DESCRIPTION
**Pull request by afuetterer**
## Description

This change offers a tiny performance boost in test collection time, as it reduces the number of directory traversals.

## Types of Changes
- [x] Other (please describe):

## Checklist
- [x] I have read the [contributor guide](https://github.com/rdmorganiser/rdmo/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -188,8 +188,7 @@ rest_framework = ["rest_framework"]
 
 [tool.pytest.ini_options]
 DJANGO_SETTINGS_MODULE = "config.settings"
-testpaths = ["rdmo"]
-python_files = "test_*.py"
```
Comment (afuetterer) [inline pyproject.toml:192]: `test_*.py` is the default behaviour in pytest test discovery.

Comment (MyPyDavid) [reply pyproject.toml:192]: Are we sure that all the tests are still discovered? 😅 
Last time I made a change to something with test paths, I accidentally excluded some tests.
I've checked now and all `23764 passed`.
Do you think we should add the pytest and coverage summaries to the Action summary? I can open another Issue and PR for it..

Comment (afuetterer) [reply pyproject.toml:192]: You can use `pytest --collect-only` for that.

With
```toml
testpaths = ["rdmo/*/tests"]
```

```console
$ pytest --collect-only
23764/23821 tests collected (57 deselected)
```

With
```toml
testpaths = ["rdmo"]
python_files = "test_*.py"
```

```console
$ pytest --collect-only
23764/23821 tests collected (57 deselected)
```

Comment (afuetterer) [reply pyproject.toml:192]: So, yes, I am positive that all tests are collected.

Comment (MyPyDavid) [reply pyproject.toml:192]: yes the coveralls was doing that, but I think only for PRs into `main`. 
So the PR dev-2.20 has 
> Coveralls - postgres: 3.12 — Coverage decreased (-0.8%) to 90.785%   

in the checks overview

**Review by MyPyDavid — APPROVED** [review]
all tests are included still 🙏

File: pyproject.toml (hunk 1 above)
Comment (afuetterer) [reply pyproject.toml:192]: Maybe there is a setting for that in coveralls? Add comments for all pull requests, not only PR to master/main?

**Pull request merged**